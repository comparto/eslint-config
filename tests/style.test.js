const style = require('../src/rules/style')

test('style.js', () => {
  expect(style).toMatchSnapshot()
})
