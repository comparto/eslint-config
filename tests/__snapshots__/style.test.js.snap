// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`style.js should create default configuration 1`] = `
Object {
  "extends": Array [
    "plugin:prettier/recommended",
    "prettier/react",
    "prettier/@typescript-eslint",
    "plugin:jest/style",
  ],
  "plugins": Array [
    "prettier",
    "jest",
  ],
  "rules": Object {
    "prettier/prettier": "error",
  },
}
`;
